concourse/tasks/deploy-to-govuk-paas.yml

```diff
@@ -24,3 +24,5 @@ run:
       cf auth
       cf t -o "$CF_ORG" -s "$CF_SPACE"
       cf push $CF_APP_NAME --strategy rolling
+
+      cf map-route $CF_APP_NAME "$DOMAIN" --hostname www
```

Comment:
>Seems the route mapping happens after push, which is where concourse is freaking out right now...

This can be solved by [specifying the domain when doing the push](https://docs.cloudfoundry.org/devguide/deploy-apps/deploy-app.html#custom-route), e.g.
```
cf push $CF_APP_NAME --strategy rolling --hostname www -d $DOMAIN
```

Reply:
ooh very fancy!
Thanks once again, highly apprecaite your `cf`-fu :D

Reply:
[amended](https://github.com/alphagov/govuk-account-static-errors/commit/d523ab7b6d1ff168a481b54cc88ea20577dd13d0)